packages/cli/src/databases/repositories/sharedWorkflow.repository.ts

```diff
@@ -22,6 +22,14 @@ export class SharedWorkflowRepository extends Repository<SharedWorkflow> {
 		return await this.exist({ where });
 	}
 
+	async getSharedUserIds(workflowId: string) {
+		const sharedWorkflows = await this.find({
+			select: ['userId'],
+			where: { workflowId },
+		});
+		return sharedWorkflows.map((sharing) => sharing.userId);
```

Comment:
When we switch to our fork, I'd love an option to return an array of what we specifically `select`ed instead of having to pick it out from the result every time.